python/pyspark/sql/tests/test_utils.py

```diff
@@ -729,46 +729,6 @@ def test_diff_schema_lens(self):
             message_parameters={"df_schema": df1.schema, "expected_schema": df2.schema},
         )
 
-    def test_assert_equal_maptype(self):
```

Comment:
Shall we keep and update this test to see whether the change on `sort` works for map type or not? If not, we should still show some error.